spec/features/visitor_views_neighborhood_details_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+feature 'visitor views neighborgood details', %(
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.